contracts/mainnet/connectors/uniswapV3/helpers.sol

```diff
@@ -0,0 +1,171 @@
+pragma solidity ^0.7.5;
+pragma abicoder v2;
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {DSMath} from "../../common/math.sol";
+import {Basic} from "../../common/basic.sol";
+import {INonfungiblePositionManager, ISwapRouter} from "./interface.sol";
+import "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+import "@uniswap/v3-core/contracts/libraries/TickMath.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol";
+import "@uniswap/v3-periphery/contracts/base/LiquidityManagement.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    /**
+     * @dev uniswap v3 NFT Position Manager & Swap Router
+     */
+    INonfungiblePositionManager nftManager =
```

Comment:
@cryptoDev222 variables should be constant variable. Storage variables of connectors are not read during delegateCall

Reply:
Okay, got it.